build.gradle

```diff
@@ -19,16 +19,13 @@ buildscript {
         maven { url "https://maven-central.storage.googleapis.com"}
```

Comment:
This CL needs some changes if you still plan on merging, these lines are probably unnecessary, gradle pulls in all plugin dependencies when using the new format from jCenter. I would probably just rewrite it if you guys wanted to actually do this.